utils.py

```diff
@@ -20,3 +21,51 @@ def log(self, string):
         if self._enabled:
             prefix = "[terminal_view debug] [%.3f] " % (time.time())
             print(prefix + string)
+
+
+class TerminalViewManager():
+    """
+    A manager to control all TerminalView instance.
+    """
+    @classmethod
+    def register(cls, terminal_view):
+        if not hasattr(cls, "terminal_views"):
+            cls.terminal_views = {}
+        cls.terminal_views[terminal_view.view.id()] = terminal_view
+        return terminal_view
+
+    @classmethod
+    def load_from_id(cls, vid):
+        if vid in cls.terminal_views:
+            return cls.terminal_views[vid]
+        else:
+            raise Exception("terminal view not found.")
+
+
+class TerminalViewSendString(sublime_plugin.WindowCommand):
```

Comment:
I'm using this branch now. Cool stuff!

Why not just turn this into a TextCommand and do `terminal_view = TerminalViewManager.load_from_id(self.view.id())` right from the beginning of the run method?